### DISCUSSION TITLE
adding saml based accounts as siteadmin

### DISCUSSION THREAD

**Discussion by rnadupal** [community/community#66909] [Enterprise]
Hello, 

I am getting the below message while adding users to the site admin role.  I've checked the GitHub docs but didn't find a way to enable this. I am using GH Enterprise, and not using scim provisioning. 

`=============
Add owners by setting their SAML administrator attribute to true.
=============
![saml](https://github.com/community/community/assets/142362564/da220a4a-9c5a-4dc6-8b00-9907fc189237)
`

Any suggestions would be greatly appreciated.

**Comment by queenofcorgis** [comment]
Hi there - to make sure I'm getting you the best info do you mind sharing which Enterprise instance you're using (Enterprise Cloud, Enterprise Server)
